commands/Factorio/amilinked.js

```diff
@@ -0,0 +1,32 @@
+const { MessageEmbed } = require("discord.js");
+const Command = require("../../base/Command.js");
+
+class Linkme extends Command {
+  constructor(client) {
+    super(client, {
+      name: "amilinked",
+      description: "Check linking status of a user",
+      usage: "<UserID>",
```

Comment:
The convention for usage in comfyV2 is square brackets for required info, and round brackets for optional info